tests/sparsecca/sparsecca/_multicca_lp.py

```diff
@@ -0,0 +1,172 @@
+import numpy as np
+from scipy.linalg import svd
+import pyomo.environ as pyo
+from collections import defaultdict
+#from sparsecca import _utils_pmd
+
+from ._utils_pmd import binary_search, l2n, soft, scale
+
+import pandas as pd
+
+   
+
+# Linear Programming 
+
+
+def preprocess_datasets(datasets:list, standardize=True, mimic_R=True):
```

Comment:
I mentioned this already, but this is copy and pasted :.( if you find yourself using something from `_multicca_pmd`, that just means that function should be a utility function and it should be moved accordingly.